fiat-ldap/src/main/java/com/netflix/spinnaker/fiat/roles/ldap/LdapUserRolesProvider.java

```diff
@@ -93,12 +99,55 @@
         .collect(Collectors.toList());
   }
 
+  private class GroupToUsers implements AttributesMapper<Pair<String, List<String>>> {
+    public Pair<String, List<String>> mapFromAttributes(Attributes attrs) throws NamingException {
+      String group = attrs.get(configProps.getGroupRoleAttributes()).get().toString();
+      NamingEnumeration<?> members = attrs.get(configProps.getGroupsUserAttributes()).getAll();
+      List<String> member = new ArrayList<>();
+      while (members.hasMore()) {
+        try {
+          String user =
+              Arrays.stream(configProps.getUserDnPattern().parse(members.next().toString()))
```

Comment:
why can't we simply cast `Object[] [0]` to String here? seems to yield cleaner code